src/screens/FeaturedEventListScreen/component.js

```diff
@@ -16,9 +22,29 @@ type Props = {
 };
 
 class EventsScreen extends PureComponent<Props> {
+  static navigationOptions = {
+    header: null,
+    tabBarVisible: false
+  };
+
   render() {
     return (
       <View style={styles.container}>
+        <Header>
+          <ContentPadding style={styles.headerContent}>
+            <IconButton
+              accessibilityLabel="Back"
+              onPress={() => {
+                this.props.navigation.goBack(null);
+              }}
+              source={chevronLeftWhite}
+            />
+            <Text type="h2" style={styles.headerTitle}>
+              {text.featuredEventListTitle}
+            </Text>
+            <View style={styles.phantomIcon} />
```

Comment:
It just adds a invisible icon space on the right so that we can just use flex box space-between to align everything. Was the simplest solution I could think of.